index.yaml

```diff
@@ -86,3 +86,8 @@ indexes:
   - name: week_start
   - name: sent_count
     direction: desc
+
+- kind: LoveCount
+  ancestor: yes
+  properties:
+  - name: week_start
```

Comment:
Was this auto added? Strange that my initial query didn't add one but once you added the `week_start` filter it picked it up 🤔

Reply:
This was auto added.